CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## Version 3.2.7
-* Added parameter in core section of nginx.conf via pillar
+* 3 nginx.conf parameters can be customised via pillar
```

Comment:
If its a priority it can be done by a releaser very quickly, lets not shortcut the procedure, that way madness lies!